README.md

```diff
@@ -15,54 +15,59 @@ Now:
 * add expRESTive to your _package.json_ file: `$ npm install --save exprestive`
 * add the expRESTive middleware to your application
 
-    ```coffeescript
-    express = require 'express'
-    exprestive = require 'exprestive'
+    ```javascript
+    express = require('express')
+    exprestive = require('exprestive')
 
     app = express()
-    app.use exprestive()
+    app.use(exprestive())
 
-    app.listen 3000
+    app.listen(3000)
     ```
 
-* create a `routes.coffee` file in the same directory as your server
+* create a `routes.js` file in the same directory as your server
```

Comment:
Do you want to mention than `routes.coffee` or `routes.ls` are also valid options here?